index.html

```diff
@@ -6,23 +6,51 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>memory game</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
     <script src="index.js" defer></script>
     <header>
-      <img src="images/logo.jpg" height="50px" width="50px" alt="logo" class="logo">
-      <h1>memory game</h1>
-      <button id="restart-btn" onclick="restartGame()">restart</button>
+      <!-- left wrapper contain logo and game name  -->
+      <div class="left-wrapper">
+        <img src="images/logo.jpg" height="50px" width="50px" alt="logo" class="logo">
+        <h1 style="margin-left:10px;">memory game</h1>
+      </div>
+      <!-- Right Wrapper for start and restart button  -->
+      <div class="right-wrapper">
+        <button id="start-btn" onclick="openModel()">Start Game</button>
+        <button id="restart-btn" onclick="restartGame()">restart</button>
+      </div>
+     
     </header>
+     <!-- Model pop-up box start here -->
+     <div class="model-container" id="model">
+        <div class="model">
+           <img src="images/logo.jpg" height="70px" width="70px" alt="logo" class="logo">
+           <h3>Choose Level</h3>
+           <!-- Levels section  -->
+           <div class="levels">
+             <button onclick="setTimer(),closeModel()">Easy</button>
+             <button onclick="setTimer(),closeModel()">Medium</button>
+             <button onclick="setTimer(),closeModel()">Hard</button>
+           </div>
+           <!-- Levels section end -->
+           <i class="fas fa-times" onclick="closeModel()"></i>
```

Comment:
no need to call multiple functions from the button on click. Just call a generic function and pass level as its parameter